pallets/interest-accrual/src/lib.rs

```diff
@@ -433,7 +433,7 @@ pub mod pallet {
 		pub(crate) fn validate_rate(interest_rate_per_year: T::InterestRate) -> DispatchResult {
 			let four_decimals = T::InterestRate::saturating_from_integer(10000);
 			ensure!(
-				interest_rate_per_year < One::one()
+				interest_rate_per_year <= One::one()
```

Comment:
[Slack thread](https://kflabs.slack.com/archives/C04HJN404RK/p1677153843734989).

- Using 0% means no gain for the system. The user should pay the same amount of the collateral no matter when it repays. It could not be practical for any real investor, but for testing, it's could be helpful to have a reference loan that does not accrual.
- Also, I do not see why we need to set an upper limit, but I let it as it is.